### PR TITLE
Update strings_uk.arb

### PR DESCRIPTION
**Pull request by Andygol**

### CODE REVIEW

File: strings/strings_uk.arb [hunk 1]
```diff
@@ -334,5 +334,9 @@
   "threadgateViewFollowing": "Користувачі за якими стежить {username}",
   "threadgateViewList": "Користувачі зі списку \"{listName}\"",
   "settingsPanelModeration": "Модерація",
-  "settingsSound": "Обрати звук сповіщення"
-}
\ No newline at end of file
+  "settingsSound": "Обрати звук сповіщення",
+  "scheduleInvalidAccount": "Ви намагалися запланувати публікацію за допомогою облікового запису, що немає доступу до планування, будь ласка, налаштуйте його.",
+  "confirmRetryScheduledPost": "Повторити заплановану публікацію",
+  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося надіслати, переконайтеся, що ви належним чином привʼязали свій обліковий запис публікацій за розкладом, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі "в минулому". Ви впевнені, що хочете, повторити ця публікацію?",
```
Comment (Andygol) [inline strings/strings_uk.arb:340]:
```suggestion
  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося надіслати, переконайтеся, що ви належним чином привʼязали свій обліковий запис публікацій за розкладом, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі «в минулому». Ви впевнені, що хочете, повторити ця публікацію?",
```

**Comment by Andygol** [comment]
@gildaswise sure, I'll do a little bit later

**Comment by Andygol** [comment]
@gildaswise I've added translation for the rest of strings. Could you take a look and merge this PR? 🙏

**Review by gildaswise — APPROVED** [review]

File: strings/strings_uk.arb [hunk 2]
```diff
@@ -334,5 +334,11 @@
   "threadgateViewFollowing": "Користувачі за якими стежить {username}",
   "threadgateViewList": "Користувачі зі списку \"{listName}\"",
   "settingsPanelModeration": "Модерація",
-  "settingsSound": "Обрати звук сповіщення"
-}
\ No newline at end of file
+  "settingsSound": "Обрати звук сповіщення",
+  "scheduleInvalidAccount": "Ви намагалися запланувати публікацію за допомогою облікового запису, що немає доступу до планування, будь ласка, налаштуйте його.",
+  "confirmRetryScheduledPost": "Повторити заплановану публікацію",
+  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося виконати, переконайтеся, що ви належним чином привʼязали свій обліковий запис для запланованих дописів, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі «в минулому». Ви впевнені, що хочете, повторити ця публікацію?",
```
Comment (Andygol) [inline strings/strings_uk.arb:340]:
```suggestion
  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося виконати, переконайтеся, що ви належним чином привʼязали свій обліковий запис для запланованих дописів, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі «в минулому». Ви впевнені, що хочете, повторити цю публікацію?",
```

Comment (gildaswise) [reply strings/strings_uk.arb:340]: Oh, did you mean to add this change? I believe you would need to open another PR since I merged this one :(

**Pull request merged**